install.sh

```diff
@@ -4,9 +4,11 @@
 
 set -e
 
+win_target="x86_64-pc-windows-msvc"
 case $(uname -s) in
 Darwin) target="x86_64-apple-darwin" ;;
-*) target="x86_64-unknown-linux-gnu" ;;
+Linux) target="x86_64-unknown-linux-gnu" ;;
+*) target=$win_target ;;
```

Comment:
I don't think the default target should be Windows. Wikipedia has a long list of `uname -s` output: https://en.wikipedia.org/wiki/Uname For Linux, there's `FreeBSD`, `GNU`, `DragonFly`. For Windows, Wine uses `Windows_NT`, Cygwin uses `CYGWIN_NT-$version`, MingW uses `MINGW$arch_NT-$version`, MSYS uses `MSYS_NT-$version`.

Maybe `$OS = "Windows_NT"` would be a good check as well? (see: https://en.wikipedia.org/wiki/Environment_variable#Windows)

Reply:
Would it be okay to check for `$OS = "Windows_NT"` then if not Darwin, then default to Linux?